iot-e2e-tests/android/app/src/androidTest/java/com/microsoft/azure/sdk/iot/android/iothub/FileUploadAndroidRunner.java

```diff
@@ -36,13 +37,4 @@ public FileUploadAndroidRunner(IotHubClientProtocol protocol, AuthenticationType
     {
         super(protocol, authenticationType, withProxy);
     }
-
```

Comment:
Do we still need these android runners? Is it for separating tests by test groups?

Reply:
At this point, yes, they only exist so that we can run them in smaller groups